From bd28936145196e9d6e13e4f2ee6b3ded7b647a6a Mon Sep 17 00:00:00 2001
From: Zhao Qiang <B45475@freescale.com>
Date: Thu, 21 Nov 2013 14:10:49 +0800
Subject: [PATCH 318/383] powerpc/p1010rdb:update dts for pcie interrupt-map

p1010rdb-pb use the irq[4:5] for inta and intb to pcie,
it is active-high, so set it.

Signed-off-by: Zhao Qiang <B45475@freescale.com>
Change-Id: I012fb7a7333966a94332040c425577e16b10b927
Reviewed-on: http://git.am.freescale.net:8181/6777
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Tiefei Zang <roy.zang@freescale.com>
Reviewed-by: Thomas Trefny <Tom.Trefny@freescale.com>
[Original patch taken from QorIQ-SDK-V1.5-SOURCE-20131219-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/powerpc/boot/dts/fsl/p1010si-post.dtsi |    8 ++++++--
 1 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/p1010si-post.dtsi b/arch/powerpc/boot/dts/fsl/p1010si-post.dtsi
index 1804ed7..4313ff6 100644
--- a/arch/powerpc/boot/dts/fsl/p1010si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p1010si-post.dtsi
@@ -59,8 +59,12 @@
 		interrupt-map-mask = <0xf800 0 0 7>;
 		interrupt-map = <
 			/* IDSEL 0x0 */
-			0000 0x0 0x0 0x1 &mpic 0x4 0x1 0x0 0x0
-			0000 0x0 0x0 0x2 &mpic 0x5 0x1 0x0 0x0
+			/*
+			 *irq[4:5] only for PCIe in dts, set to active-high
+			 *irq[6:7] is pulled up on board, set to active-low
+			 */
+			0000 0x0 0x0 0x1 &mpic 0x4 0x2 0x0 0x0
+			0000 0x0 0x0 0x2 &mpic 0x5 0x2 0x0 0x0
 			0000 0x0 0x0 0x3 &mpic 0x6 0x1 0x0 0x0
 			0000 0x0 0x0 0x4 &mpic 0x7 0x1 0x0 0x0
 			>;
-- 
1.7.5.4

